ios/FluentUI/Bottom Commanding/BottomCommandingController.swift

```diff
@@ -294,6 +294,8 @@ open class BottomCommandingController: UIViewController {
 
         // Make sure we only reset our commanding style on iPad. On iPhone we only use the sheet style, so there's no need for unnecessary work.
         if newCollection.horizontalSizeClass != traitCollection.horizontalSizeClass && newCollection.userInterfaceIdiom == .pad {
```

Comment:
Just checking, but this changed line is scoped to iPad (e.g. -- .pad) which means that it doesn't impact iPhone, which is what we want?
Also, I'm assuming then that you are verifying on both devices (iPhone and iPad) as the validation didn't exactly specify.

Reply:
Correct, I'll update the validation info. We only ever want to dismiss popovers on iPad. We never even present anything on iPhone.